eng/Version.Details.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.TemplateEngine.Abstractions" Version="8.0.300-preview.24206.1">
+    <Dependency Name="Microsoft.TemplateEngine.Abstractions" Version="8.0.300">
```

Comment:
@dotnet/source-build-internal 

Looks like SourceBuild is expecting this version but can't find it. 

> ##[error].packages/microsoft.dotnet.arcade.sdk/8.0.0-beta.24204.3/tools/Tools.proj(0,0): error NU1102: (NETCORE_ENGINEERING_TELEMETRY=Restore) Unable to find package Microsoft.SourceBuild.Intermediate.templating with version (= 8.0.300)

https://dev.azure.com/dnceng-public/public/_build/results?buildId=650026&view=logs&jobId=2f0d093c-1064-5c86-fc5b-b7b1eca8e66a&j=2f0d093c-1064-5c86-fc5b-b7b1eca8e66a&t=d2b639a6-cb19-5f1f-66fd-8047f66b3026